src/sidebar/components/Annotation/AnnotationEditor.tsx

```diff
@@ -167,6 +167,9 @@ function AnnotationEditor({
 
   const textStyle = applyTheme(['annotationFontFamily'], settings);
 
+  const atMentionsEnabled = store.isFeatureEnabled('at_mentions');
```

Comment:
This is fine, `mentionsEnabled` would also work as a slightly more succinct option.